baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -589,7 +591,9 @@ def import_snapshot(self):
                     return
                 elif self.config["snapshot_mode"] == "file":
                     self.query_step(snapshot_file_query)
-                    snapshot_file = self.config["snapshot_file"]
+                    snapshot_file = os.path.join(TMP_SNAPSHOT_LOCATION, "file.snapshot")
+                    # not copying since it can take a lot of time
+                    os.link(self.config["snapshot_file"], snapshot_file)
```

Comment:
Should we also remove this symlink after all? Consider the following steps:

1. User imported the snapshot from a file (in my case, I cancelled the import, but I believe it doesn't matter)
2. User runs `tezos-setup` once again and imports the same file

After it user will get the following error

```
Traceback (most recent call last):
  File "src/tezos_baking/tezos_setup_wizard.py", line 839, in main
    setup.run_setup()
  File "src/tezos_baking/tezos_setup_wizard.py", line 783, in run_setup
    self.bootstrap_node()
  File "src/tezos_baking/tezos_setup_wizard.py", line 654, in bootstrap_node
    self.import_snapshot()
  File "src/tezos_baking/tezos_setup_wizard.py", line 596, in import_snapshot
    os.link(self.config["snapshot_file"], snapshot_file)
FileExistsError: [Errno 17] File exists: '/home/dmitriy/serokell/node-snapshots/tezos-mainnet.rolling' -> '/tmp/octez_node.snapshot.d/file.snapshot'
```

Reply:
It does matter, since `/tmp/octez_node.snapshot.d/` gets cleaned up after import, but it is still a fair point, it shouldn't fail here